howto/android/system-tracing.md

```diff
@@ -0,0 +1,48 @@
+This guide describes how to enable system tracing in Android using Anbox Cloud. 
+
+## Enable Android System Tracing
+
+1. To enable Android system tracing, add the `android.enable_tracing` flag into the feature file. In the Android container, run:
```

Comment:
> In the Android container

I changed the "development container" to "Android container". Is it correct? Or should this be "In the Anbox instance"?

Reply:
Android runs in a container nested in an instance (container or VM), so that seems accurate to me.